rl4j/rl4j-core/src/main/java/org/deeplearning4j/rl4j/util/VideoRecorder.java

```diff
@@ -100,6 +87,7 @@ public void startRecording(String filename) throws Exception {
 
         fmpegFrameRecorder = new FFmpegFrameRecorder(filename, width, height);
         fmpegFrameRecorder.setVideoCodec(codec);
+        //fmpegFrameRecorder.setPixelFormat(AV_PIX_FMT_RGB8);
```

Comment:
Please remove commented-out code, add a fixme, or add a comment saying why the line is commented-out.

Reply:
Let's just make it another parameter?